lib/src/components/buttons/base_button.dart

```diff
@@ -67,4 +71,8 @@ abstract class DSFRBaseButton extends StatelessWidget {
   final IconPosition iconPosition;
 
   final bool iconOnly;
+
+  final MainAxisSize? mainAxisSize;
 }
+
+mixin GroupeableButton on DSFRBaseButton {}
```

Comment:
This should be an interface instead of a mixin. In this case you want to specify that primary and secondary button are `GroupeableButton`, but you don't implement any logic. 

[From an interresting SO post](https://stackoverflow.com/questions/45901297/when-to-use-mixins-and-when-to-use-interfaces-in-dart)

> Mixins is all about how a class does what it does, it's inheriting and sharing concrete implementation. Interfaces is all about what a class is, it is the abstract signature and promises that the class must satisfy. It's a type.